lib/factree.rb

```diff
@@ -1,5 +1,9 @@
 require "factree/version"
+require "factree/conclusion"
+require "factree/path"
+require "factree/aggregate"
+require "factree/dsl"
```

Comment:
Generally we do `autoload` over explicit requires now; e.g. https://github.com/ConsultingMD/schlepper/blob/master/lib/schlepper.rb#L7